<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="CB_DevList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Lbl_PortNum">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Port Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_PortNum">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1234</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lbl_SrcIP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Source IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_SrcIP">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.128</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_SetNetwork">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Network</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_Start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Logging</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lbl_FrameLen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame Length</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_FrameLen">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1024</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lbl_Pattern">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sync Pattern(hex)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Pattern">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>feef</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lbl_Fragment">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fragment</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_Fragment">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>64</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>402</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
